hail/src/main/scala/is/hail/linalg/BlockMatrix.scala

```diff
@@ -415,16 +415,16 @@ class BlockMatrix(val blocks: RDD[((Int, Int), BDM[Double])],
   val nCols: Long) extends Serializable {
 
   import BlockMatrix._
-  
+
```

Comment:
if it's not a ton of trouble, would you mind reverting the whitespace changes here? Might cause rebase trouble with the other BM churn happening right now with lowering.